README.md

```diff
@@ -46,23 +46,24 @@ A template Service Approval Accelerator is maintained [here](templates/ServiceAp
 
 This WG exists to develop, maintain, and document easily consumable infrastructure as code (IaC) which can be used as a base for deploying systems in highly-regulated environments.
 
-Detailed documentation regarding the process for developing and delivering IaC can be found [here](?).
+Detailed documentation regarding the process for developing and delivering IaC can be found [here](https://github.com/finos/compliant-financial-infrastructure/blob/docs/wg-readme/docs/terraform-module-best-practices.md).
 
 ### High level objectives
 
-- TODO
-
+- Understand the policies that need to be deployed via IaC for supported Cloud Services (Input)
```

Comment:
We could probably just replace "cloud services" with "services" or "service providers" here and cover basically everything, it would probably make the sentence a bit meaningless though?

Reply:
Removed the first bullet here